test/src/module/command/archivePushTest.c

```diff
@@ -394,11 +458,14 @@ testRun(void)
         // -------------------------------------------------------------------------------------------------------------------------
         TEST_RESULT_BOOL(archivePushProtocol(strNew(BOGUS_STR), paramList, server), false, "invalid function");
 
-        // Create a new encrypted repo to test encryption
         // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("multiple repos, one encrypted");
```

Comment:
We already test compression elsewhere in this module and I'd rather not add more compression tests since they add so much time to the test.

Better verification of the files sounds like a good idea. repo-get is too slow for this purpose (because it can't read a file into memory) but it would be simple enough to write a function in the module to do this.

Reply:
Per discussion, since Mock and Real tests cover file contents checking, it was agreed that we would do this later in the unit tests when Mock are pared down/removed.